### DISCUSSION TITLE
Allow comments directly on the question without requiring an Answer to be submitted

### DISCUSSION THREAD

**Discussion by tonygermano** [community/community#3798] [Discussions] [answered]
Often times the posted question leads to more questions asking for clarification. It would be useful to post comments requesting clarification directly on the question, rather than having to submit them as an Answer, when they are clearly not intended to be one. Answers accept comments, and the original question should as well.

**Comment by benhaynes** [comment, marked as answer]
Just like StackOverflow. Also, this is not an answer, this is just a comment. 😉
